### DISCUSSION TITLE
calendar link icon should this be different

### DISCUSSION THREAD

**Discussion by marksie1988** [totaldebug/atomic-calendar-revive#1129] [Polls]
Hi All, 

![image](https://github.com/totaldebug/atomic-calendar-revive/assets/159166/9d02be82-2a75-49cc-9ee5-3fd8399afe5c)

The above icon currently links to google calendar by default, obviously for people using caldav or multiple calendars this button is pointless. 

I am proposing making a change to this but am unsure which the users would prefer: 

1. Add an option to add a customCalendarLink
  a. This would allow adding a single URL of your choosing
2. Add a mode switcher similar to the native calendar
  a. this would allow switching between event mode and calendar mode
  b. i would add an option to disable this too 
  c. I could also add a button to add events to the same ribbon  (this would need investigation)
3. remove it all together no one uses it
4. add a button that allows adding events (this would need investigation) 

if you have any other preferences that I haven't accounted for please comment and provide some feedback

**Comment by JTP335d** [comment]
I use iOS and caldav and would like to use that button as a tap action for navigate or url. Specifically url 'Calendar://' to open the iOS calendar app directly instead of google calendar in a browser.
